redaxo/src/addons/backup/lib/backup.php

```diff
@@ -221,7 +221,7 @@ public static function importFiles($filename)
         self::importScript(str_replace('.tar.gz', '.php', $filename), self::IMPORT_ARCHIVE, self::IMPORT_EVENT_PRE);
 
         $tar->openTAR($filename);
-        if (!$tar->extractTar()) {
+        if (!$tar->extractTar(rex_path::base())) {
```

Comment:
ich verwende hier jetzt fest den `rex_path::base`-pfad.

wenn man den media-path umdefiniert hat, könnte das ggf. dazu führen dass beim export die dateien nicht an der richtigen stelle landen. mich wundert ob das vorher überhaupt schonmal sinnvoll funktioniert hat mit wenn man einen eigenen path-provider hat.

da sollte jemand von yakamara vermutlich was dazu sagen können

Reply:
Ich glaube, wir nutzen die Dateibackups nicht.

Aber ich kapiere noch nicht ganz: Wird es durch deine Änderung hier besser, oder schlechter, oder ändert sich an dem Aspekt nix?

Reply:
Ich denke nicht dass sich was im bezug af vorher ändert.

Mir war/ist nicht bewusst woher die alte tar klasse wusste in welches verzeichnis exportiert werden soll.. konnte keine pfad angabe dazu finden

Reply:
Es arbeitete scheinbar mit relativen Pfaden zum Projektroot, soweit ich es verstehe.